### DISCUSSION TITLE
Projects subtenant

### DISCUSSION THREAD

**Discussion by hdirki** [andrewdwallo/filament-companies#112] [General]
What would be the best logical approach to have a Project model which is a subtenant of Company?

For example, with a Company, the Company Owner can create multiple projects within the Company and assign specific Company Employees to the project. The project would have its own standalone panel and is tenanted (i.e. shows data for that project only). Similarly, there would be a Project Switch.

Popular example include Asana (you can create multiple Workspaces, i.e. Companies). Within each Workspace, you can create projects and assign Workspace Users to each project.

Hope this makes sense?

**Comment by andrewdwallo** [comment]
You would need to make sure you have the appropriate relationships for that first. Then you could create a role/permission that an owner could assign to the users. This role/permission would check whether that user could both see and operate within a project.
